### PR TITLE
[Internal] Query:  Fixes and enables hybrid search tests and configures pipeline to use internal emulator.

### PR DESCRIPTION
**Pull request by sc978345**
## Description
This PR modified the emulator setup to use the internal emulator build, and sets queryEnableFullText to true so that we can test full text search

It also enables some full-text-search-related tests

### CODE REVIEW

**Review by kundadebdatta — DISMISSED** [review]
LGTM!

**Comment by adityasa** [comment]
It's not clear why following tests were marked as ignore in Debdatta's earlier pr (I don't see Full-Text dependency but I may be missing something). Can you please double check? @kundadebdatta in case he knows:
CosmosContainerSettingsTests.DefaultIndexingPolicySameAsDocumentCollection
SettingsContractTests.ContainerV2CompatTest

**Review by adityasa — DISMISSED** [review]
:shipit:

**Review by neildsh — APPROVED** [review]

File: templates/emulator-setup.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 steps:
   - pwsh: |
-      Write-Host "Downloading Cosmos Emulator - $env:EMULATORMSIURL" -ForegroundColor green
```
Comment (sourabh1007) [inline templates/emulator-setup.yml:5]: is it temporary change or permanent? and why are we not configuring it, using environment variable?

Comment (kirankumarkolli) [reply templates/emulator-setup.yml]: Temporary and will be reverted in future. 
This change was moved to a different PR which is merged

**Pull request merged**